Gemfile.lock

```diff
@@ -216,7 +217,7 @@ GEM
     listen (3.7.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.16.0)
+    loofah (2.19.0)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.16.0 → 2.19.0

[_[change-log](https://github.com/flavorjones/loofah/blob/main/CHANGELOG.md), [source-code](https://github.com/flavorjones/loofah), [gem-diff](https://my.diffend.io/gems/loofah/2.16.0/2.19.0)_]





<details>
<summary>Commits</summary>

A change of **16** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.16.0...v2.19.0).

These are the first 10 commits:
- (8d183d3) [Adds ARIA attributes](https://github.com/flavorjones/loofah/commit/8d183d376815eab6849eaa058ff558b6e1620292)
- (5f58b9c) [feat: add the remaining aria attributes and an ad-hoc test](https://github.com/flavorjones/loofah/commit/5f58b9cd3d0d4731181e20bd589818da5a20289b)
- (2fc3bfb) [Merge pull request #233 from flavorjones/flavorjones-aria-at…](https://github.com/flavorjones/loofah/commit/2fc3bfb7ba00f384ef23c54264c271fbe6381bac)
- (22fcdda) [version bump to 2.17.0](https://github.com/flavorjones/loofah/commit/22fcddaf691928eeed949a8b7fc82a71d50f77f7)
- (303ff07) [test: ensure we pass with libxml 2.9.14](https://github.com/flavorjones/loofah/commit/303ff078afb334f94b09c5d466606b0131f85d4a)
- (c8ae0a4) [Merge pull request #235 from flavorjones/flavorjones-update-…](https://github.com/flavorjones/loofah/commit/c8ae0a4270abdacdb2a4590c5be50c5395f5d5eb)
- (d3e82c1) [update README to remove pledgie link](https://github.com/flavorjones/loofah/commit/d3e82c1eadde60b16e5178b17c5dad057cb12937)
- (8a4d85d) [Add aspect-ratio to the list of css properties](https://github.com/flavorjones/loofah/commit/8a4d85d291427934c63f5b701d2c1c0a36d93d1a)
- (4963914) [Merge pull request #236 from louim/patch-1](https://github.com/flavorjones/loofah/commit/496391482c6c038e814e1154500a220c5b0fc7dd)
- (0c08ea5) [ci: do not fail fast](https://github.com/flavorjones/loofah/commit/0c08ea59e2d61c0aaafe946dd79442241888b5bd)


</details>